test/Test/Toml/Codec/Combinator/Custom.hs

```diff
@@ -0,0 +1,21 @@
+module Test.Toml.Codec.Combinator.Custom
+    ( customSpec
+    ) where
+
+import Test.Hspec (Spec, describe)
+
+import Test.Toml.Codec.Combinator.Common (codecRoundtrip)
+
+import qualified Hedgehog.Gen as Gen
+import qualified Test.Toml.Gen as Gen
+import qualified Toml.Codec.BiMap.Conversion as Toml
+import qualified Toml.Codec.Combinator.Custom as Toml
+
+
+customSpec :: Spec
+customSpec = describe "Combinator.Custom: Roundtrip tests" $ do
+    codecRoundtrip "Read (Int)      " (Toml.read @Int) Gen.genInt
```

Comment:
Maybe we can add "dummy" `textBy` check as well?
Something like `textBy show (maybeToEither (?) . readMaybe)`?